### PR TITLE
small variant for recommended articles block. Fixes some CSS issue fr…

### PR DESCRIPTION
**Pull request by seanchoi-workfront**
https://block-recommended-articles--small--business-website--adobe.hlx3.page/drafts/uncled/blog/kitchen-sink#recommended-articles-small

### CODE REVIEW

**Comment by fkakatie** [comment]
I think the `<a>` inside the `.recommended-articles-small-container` should be styled here in this file _or_ the appropriate class(es) could be dynamically appended by functionality in the `recommended-articles.js` file. Thoughts? 

There are some general styles for "buttons" in the `styles.css` sheet, but I couldn't find a good combination of `.secondary` or `.light` to get the same look/feel of the original blog or consonant. 

button for small rec articles

**Comment by seanchoi-workfront** [comment]
> I think the `<a>` inside the `.recommended-articles-small-container` should be styled here in this file _or_ the appropriate class(es) could be dynamically appended by functionality in the `recommended-articles.js` file. Thoughts?
> 
> There are some general styles for "buttons" in the `styles.css` sheet, but I couldn't find a good combination of `.secondary` or `.light` to get the same look/feel of the original blog or consonant.
> 
> button for small rec articles

Just realized the <a> was supposed to be button. I will fix that.

File: blocks/recommended-articles/recommended-articles.css [hunk 1]
```diff
@@ -68,8 +68,15 @@ main .recommended-articles-small-container .article-card {
...
   height: 130px;
+  border-radius: 4px 0 0 4px;
 }
+
+.recommended-articles-small-container .article-card .article-card-image img {
+  border-radius: 4px 0 0 4px;
+}
+
 .recommended-articles-small-container .article-card .article-card-body {
+  border-radius: 0 4px 4px 0;
```
Comment (fkakatie) [inline blocks/recommended-articles/recommended-articles.css]: lil adjustments for the shift from vertical to horizontal display

File: blocks/recommended-articles/recommended-articles.js [hunk 2]
```diff
@@ -4,6 +4,13 @@ import {
 } from '../../scripts/scripts.js';
 
 async function decorateRecommendedArticles(recommendedArticlesEl, paths) {
+  const articleElParent = recommendedArticlesEl.parentNode.parentNode;
+  if (articleElParent) {
+    const parentClasses = [...articleElParent.classList];
+    if (parentClasses.includes('recommended-articles-small-container')) {
+      articleElParent.querySelector('a').classList.add('button', 'small', 'action', 'light');
+    }
+  }
```
Comment (fkakatie) [inline blocks/recommended-articles/recommended-articles.js:13]: not sold on this implementation. thoughts @seanchoi-workfront?

**Review by fkakatie — APPROVED** [review]
Looking good!

**Pull request merged**